sdk/package.json

```diff
@@ -24,6 +24,7 @@
     "build_demo_win": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-%npm_package_version%.tgz && npm i",
     "build_demo_mac_lin": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-$npm_package_version.tgz && npm i",
     "run_demo": "cd ../demo && npx react-native run-android",
+    "build_no_run_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win",
     "fully_automated_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win && npm run run_demo",
     "fully_automated_mac,lin": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_mac_lin && npm run run_demo",
```

Comment:
_Future note:_ There is some duplication between tasks which would be good to deduplicate